.roo/commands/implementation-planning.md

```
@@ -0,0 +1,170 @@
+# Implementation Planning Workflow
+
+## Description
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)